e2e/suite.go

```diff
@@ -93,3 +99,18 @@ func (s *Suite) RetryOperation(operation func() error, maxRetries int) error {
 	}
 	return fmt.Errorf("operation failed after %d retries: %w", maxRetries, err)
 }
+
+func (s *Suite) K8sDefaultOptions() []k8s.Option {
+	if os.Getenv(envK8sAuthToken) == "" || os.Getenv(envK8sCACert) == "" || os.Getenv(envK8sHost) == "" {
+		s.T().Logf("%s, %s and/or %s are not set, using default cluster config from ~/.kube/config", envK8sAuthToken, envK8sCACert, envK8sHost)
+		return nil
+	}
```

Comment:
If i'm understanding the requirements here correctly, I think the logic should be if `ALL` are blank, then use the default cluster config. But if `SOME` are blank, then it should be an error that the developer didn't provider everything. 

If it in fact is OK to provide some, but not all, then the `if` condition definitely needs to be `AND` checks.